src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -2383,6 +2385,22 @@ public ContainerStats stats(final String containerId)
     }
   }
 
+  public void stats(final String containerId, final StatsHandler handler)
```

Comment:
I think this is missing an `@Overrides` annotation (I am surprised the build does not fail on that)

Reply:
Oops! Fixed.